modules/light-clients/08-wasm/internal/types/store_test.go

```diff
@@ -1,18 +1,69 @@
 package types_test
 
 import (
+	"encoding/json"
+	"errors"
+	"testing"
+
+	dbm "github.com/cosmos/cosmos-db"
+	testifysuite "github.com/stretchr/testify/suite"
+
+	"cosmossdk.io/log"
 	"cosmossdk.io/store/prefix"
 	storetypes "cosmossdk.io/store/types"
 
+	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+
+	internaltypes "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/types"
 	wasmtesting "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing"
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing/simapp"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
 	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
 	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
 	ibcexported "github.com/cosmos/ibc-go/v8/modules/core/exported"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
 var invalidPrefix = []byte("invalid/")
 
+type TypesTestSuite struct {
+	testifysuite.Suite
+	coordinator *ibctesting.Coordinator
+	chainA      *ibctesting.TestChain
+}
+
+func TestWasmTestSuite(t *testing.T) {
+	testifysuite.Run(t, new(TypesTestSuite))
+}
+
+func (suite *TypesTestSuite) SetupTest() {
+	ibctesting.DefaultTestingAppInit = setupTestingApp
+
+	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 1)
+	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(1))
+}
+
+func init() {
```

Comment:
tiny nit: I like it when `init()` is nearer the top of file, maybe underneath var definitions and above first struct definition